oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/remote/client/RemoteClientManager.java

```diff
@@ -154,7 +159,7 @@ private void printRemoteClientList() {
     }
 
     public List<RemoteClient> getRemoteClient() {
-        return usingClients;
+        return ImmutableList.copyOf(usingClients);
```

Comment:
This is heavy op. UsingClient should be immutable list itself.

Reply:
i got it, will change it